deploy/charts/cert-manager/values/deployment_strategy.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+// https://github.com/kubernetes/api/blob/b82f5c5b78f22c022611b5413297acafb79ede04/apps/v1/types.go#L320-L373
+
+// DeploymentStrategy describes how to replace existing pods with new ones.
+type DeploymentStrategy struct {
```

Comment:
This hasn't been used anywhere.
Although I see that the templates do refer to it

```sh
git grep .strategy deploy/charts/cert-manager/
deploy/charts/cert-manager/templates/cainjector-deployment.yaml:  {{- with .Values.cainjector.strategy }}
deploy/charts/cert-manager/templates/cainjector-deployment.yaml:  strategy:
deploy/charts/cert-manager/templates/deployment.yaml:  {{- with .Values.strategy }}
deploy/charts/cert-manager/templates/deployment.yaml:  strategy:
deploy/charts/cert-manager/templates/webhook-deployment.yaml:  {{- with .Values.webhook.strategy }}
deploy/charts/cert-manager/templates/webhook-deployment.yaml:  strategy:

```

But these values are not documented in the current README.

Reply:
I added the value to `deployment.go`.